apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/phasing/dao/PhasingPollTable.java

```diff
@@ -77,20 +84,47 @@ protected void save(Connection con, PhasingPoll poll) throws SQLException {
         }
     }
 
-    public DbIterator<Transaction> getFinishingTransactions(int height) throws SQLException {
+    public List<Transaction> getFinishingTransactions(int height) {
         Connection con = null;
+        List<Transaction> transactions = new ArrayList<>();
         try {
             con = getDatabaseManager().getDataSource().getConnection();
             PreparedStatement pstmt = con.prepareStatement("SELECT transaction.* FROM transaction, phasing_poll " +
                     "WHERE phasing_poll.id = transaction.id AND phasing_poll.finish_height = ? " +
                     "ORDER BY transaction.height, transaction.transaction_index"); // ASC, not DESC
             pstmt.setInt(1, height);
-            return blockchain.getTransactions(con, pstmt);
+            blockchain.getTransactions(con, pstmt).forEach(transactions::add);
+
+            return transactions;
         } catch (SQLException e) {
             DbUtils.close(con);
-            throw e;
+            LOG.error(e.getMessage(), e);
```

Comment:
Why did you supress error? When SQLException occurred you should rethrow exception to stop block push, blockchain downloading, etc